### ISSUE TITLE
Low training loss, but low map in coco when using transfer lerning?

### ISSUE THREAD

**Issue by Edwardmark** [ultralytics/yolov5#3488]
I train a rpn using coco by setting single-cls to true, and I try to fine-tune the network with num_class=80, with coco, the training loss is good, but the validataion map is quite low, I am really confused.  I freeze all layers using the following code:
```
freeze = ['model.%s.' % x for x in range(24)] # freeze backbone layer
```
Why is that?
The logs is as follows:
![image](https://user-images.githubusercontent.com/15247551/120955616-57591b00-c784-11eb-9c0e-e50743a22399.png)
But when I do not use freeze, the result is normal as follows:
![image](https://user-images.githubusercontent.com/15247551/120964329-c1c68700-c795-11eb-85d9-7f4372f78c48.png)

**Comment by Edwardmark** [comment]
@glenn-jocher Looking forward to your reply, thanks.

**Comment by glenn-jocher** [comment]
@Edwardmark do not use a bug label for questions.
